documents/motions/06.2021.1 - Regional Organizations.md

```diff
@@ -0,0 +1,57 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 6.2021.1  
+**Subject:** Regional Organizations  
+**Intent:** Procedure of acknowledgement of Regional Organizations and description of their rights and duties  
+**Submitted by:** Board of Directors  
+**Date:** February 28, 2021  
+
+# Motion
+
+The WCA shall acknowledge Regional Organizations as Members to support the Objectives of the WCA in specific regions.
+
+1. Rights of Regional Organizations:
+   1. Regional Organizations may represent the WCA in the region that they are acknowledged for, with activities that support the Objectives of the WCA.
+   2. Regional Organizations may recognize a separate organization to act on its behalf in a subsection of the region of which it covers. This organization shall have the full rights and responsibilities of the Regional Organization.
+      1. Regional Organizations must submit to the WCA a list of the Organizations which it provides authorization to.
+      2. Regional Organizations are responsible for any and all actions of organizations that act on its behalf.
+   3. Full Regional Organizations may nominate themselves to be responsible for the payment of WCA dues for the region in which they cover.
+2. Duties of Regional Organizations:
+   1. To actively organize or support WCA Competitions in the region of the organization.
+   2. To contribute to the Continental WCA Competitions that are held in the continent of the Regional Organization, if required by the corresponding Organization Team.
+   3. To respect and operationalize the Objectives set out in the WCA Bylaws and Motions.
+   4. To comply with all applicable WCA Regulations.
+   5. To accept and comply with decisions of WCA Staff.
+   6. To keep on file at the WCA a copy of its current bylaws and, if applicable, regulations in an electronic format.
+3. Requirements of Regional Organizations:
+   1. Two Regional Organizations shall not cover the same region.
+   2. Regional Organizations shall not accept sponsorship from sponsors if it would create a conflict of interest with the sponsors of the WCA.
```

Comment:
The WCA will be sure to communicate with RO's in the case of an impending sponsorship deal to work though any conflicts that may arise. It's hard to say exactly what is possible in terms of conflict of interest with future WCA sponsors but it is unlikely that anything would cause the loss of a competition sponsor without benefits to the organizer